test/gtest/ucp/test_ucp_worker.cc

```diff
@@ -936,9 +984,12 @@ class test_worker_cpu_mask : public ucp_test {
     }
 
 protected:
+    // Don't create any entities on startup
     void init() override
     {
-        // Don't create any entities on startup
+        m_rlimit = std::unique_ptr<rlimit_setter>(
```

Comment:
shall we reset it during cleanup()?
otherwise, it means it's reset during the destructor, which means it can be just a member variable.. and not need for unique_ptr

Reply:
reverted to not using unique_ptr